src/shared/components/Counter/CounterBeers.jsx

```diff
@@ -0,0 +1,10 @@
+import React, { useContext } from 'react';
+import './styles.scss';
+import { CounterBeers } from '../../../counterBeer';
+
+function CounterBeer() {
+  const { counter } = useContext(CounterBeers);
+  return <div className="counter">{counter}</div>;
+}
+
+export default CounterBeer;
```

Comment:
Remove everything that is connected with counter and change PR name.

Reply:
Done